Updated node removals docs for clarity and correctness, overuse warning

eric-solie opened the pull request:
There were several outdated items in the Node Removal guidelines. Fixed those, and hopefully made the section more clear - trying to make the node removal API more usable while discouraging overuse.

shawnrushefsky commented on products/sgs/node-removals.mdx:
````diff
@@ -39,26 +62,30 @@ The JWT will be shared with you before the trial period begins.
-## Example Request
+## Example request
+
+```Curl.sh
````
I think this should just be "shell" or "bash" to get the right highlighting

eric-solie commented:
Are there docs describing the options?

On Fri, Nov 8, 2024, 15:58 Shawn Rushefsky ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In products/sgs/node-removals.mdx
> <https://github.com/SaladTechnologies/salad-cloud-docs/pull/106#discussion_r1834539300>
> :
>
> >  </Note>
>
> -## Example Request
> +## Example request
> +
> +```Curl.sh
>
> I think this should just be "shell" or "bash" to get the right highlighting
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SaladTechnologies/salad-cloud-docs/pull/106#pullrequestreview-2424030321>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATAG2PGFCOSRK5RRU6ONEHLZ7TGTBAVCNFSM6AAAAABRNHQZI2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDIMRUGAZTAMZSGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

eric-solie commented:
@shawnrushefsky addressed your comment above. List of languages supported is visible [here ](https://prismjs.com/test.html#language=markup)for future reference.
